Uppsala/UppEt41.xml

```diff
@@ -0,0 +1,501 @@
+                                
+                                <item xml:id="e5">
+                                    <desc type="Comment">Note in pencil on the recto of the paper guard leaf 
+                                        preceding <ref target="#1r"/>.</desc>
+                                    <q xml:lang="sv">
+                                        Förkl. öfv. evangelierna<lb/>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e6">
+                                    <desc type="Comment">Note in pencil on the verso of the paper guard leaf 
+                                        preceding <ref target="#1r"/>.</desc>
+                                    <q xml:lang="sv">
+                                        Förklaring av Mattheus o Johannes Evang.<lb/>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e7">
+                                    <desc type="Comment">Typewritten note on a loose notecard between the paper guard leaf 
+                                        preceding <ref target="#1r"/> and <ref target="#1r"/> itself.</desc>
+                                    <q xml:lang="sv">
+                                        Kommentar till Matteus- o. Johannesevangeliet.<lb/>
+                                        Se Brit.Mus. Or. 735 = Wright CCCXI, där samma<lb/>
+                                        inledande teologiska traktat utan titel.
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="e8">
+                                    <desc type="AcquisitionNote">Note in pen on the notecard listed as <ref target="#e7"/>.</desc>
+                                    <q xml:lang="sv">
+                                        x 1968/31
+                                    </q>
+                                </item>
+                                
+                            </list>
+                        </additions>
+
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">European binding with quarter-leather cover.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                    </physDesc>
+                                        
+                    <history>
+                        <origin>
+                            <origDate when="1605">1605</origDate>
+                            According to a note on <locus target="#218vb"/>, the manuscript is dated to
+                            <date when="1705">1705</date> "according to the calculation of the Europeans"
```

Comment:
I think, the emendation is convincing and therefore it is probably better not to mark up the wrong date in this note.

Reply:
Good point